docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -11,13 +11,22 @@ filter on, you can safely disable indexing on this field in your
 
 [source,js]
 --------------------------------------------------
+PUT index
 {
-  "foo": {
-    "type": "integer",
-    "index": false
+  "mappings": {
+    "type": {
+      "properties": {
+        "foo": {
+          "type": "integer",
+          "index": false
+        }
+      }
+    }
   }
 }
 --------------------------------------------------
+// CONSOLE
+// TEST
```

Comment:
`// CONSOLE` implies `// TEST`. You only need `// TEST` if you need to modify the test with something like a substitution.